dev/bots/analyze_sample_code.dart

```diff
@@ -612,8 +612,17 @@ dependencies:
     sdk: flutter
 ''');
 
+
     // Copy in the analysis options from the Flutter root.
-    File(path.join(_flutterRoot,'analysis_options.yaml')).copySync(path.join(directory.path, 'analysis_options.yaml'));
+    final File rootAnalysisOptions = File(path.join(_flutterRoot,'analysis_options.yaml'));
+    final File analysisOptions = File(path.join(directory.path, 'analysis_options.yaml'));
+    analysisOptions.writeAsStringSync('''
+include: ${rootAnalysisOptions.absolute.path}
+
+analyzer:
+  errors:
+    directives_ordering: ignore
+''');
```

Comment:
what will this do in dartpad?

Reply:
It won't do anything in dartpad, which is why I filed https://github.com/flutter/flutter/issues/81697 to fix the ordering as part of publishing.

Reply:
See this PR: https://github.com/flutter/flutter/pull/81873